test/testapps/Directory.Build.targets

```diff
@@ -1,3 +1,2 @@
 <Project>
```

Comment:
Hmm, we're missing picking up the root Directory.Buid.targets (which overrides the runtime framework version). Can you have this import `SolutionRoot\Directory.Build.targets`?

Reply:
I added this but then had to take it out. It's causing some test failures and it's unrelated to my work. 

It's specifically a non-goal to make these project stay "up to date" with out build system.